Tests/FunctionalTests/EdgePathOverwriteTests.swift

```diff
@@ -186,14 +186,8 @@ class AEPEdgePathOverwriteTests: TestBase {
         mockNetworkService.assertAllNetworkRequestExpectations()
         let resultNetworkRequests = mockNetworkService.getNetworkRequestsWith(url: TestConstants.EX_EDGE_MEDIA_PROD_URL_STR, httpMethod: HttpMethod.post)
 
-        let requestPayload = resultNetworkRequests[0].connectPayload
-
-        let payload = asFlattenDictionary(data: requestPayload)
-        for key in payload.keys {
-            if key.starts(with: "events") && key.contains("request.path") {
-                XCTFail("Request object should not be sent in the edge request payload")
-            }
-        }
+        let expectedJSON = "{}"
+        assertTypeMatch(expected: expectedJSON, actual: resultNetworkRequests[0], pathOptions: KeyMustBeAbsent(paths: "events.path"))
```

Comment:
This test should check that the network request does not include the key "request" and its value under "events" since this is an internal key that Edge extension processes and removes from the event sent upstream. `events.request.path` must be absent

Reply:
Right, the intention of the original test was to check that the internal key was not added in the network request. One small correction though, I think we need to check for this path instead `events[0].request.path`.